src/modules/helpchan.ts

```diff
@@ -332,23 +319,23 @@ export class HelpChanModule extends Module {
 
 		if (!claimedChannel) {
 			return msg.channel.send(
-				'Failed to claim a help channel, no available channel.',
+				':warning: failed to claim a help channel, no available channel.',
 			);
 		}
 
 		this.busyChannels.add(claimedChannel.id);
 		const toPin = await claimedChannel.send(
 			new MessageEmbed()
-				.setAuthor(user.displayName, user.user.displayAvatarURL())
+				.setAuthor(member.displayName, member.user.displayAvatarURL())
 				.setDescription(msgContent),
 		);
 		await toPin.pin();
-		await this.addCooldown(user, claimedChannel);
+		await this.addCooldown(member, claimedChannel);
 		await this.moveChannel(claimedChannel, categories.ongoing);
 		await claimedChannel.send(
-			`<@${user.id}> this channel has been claimed for your question. Please review <#${askHelpChannelId}> for how to get help.`,
+			`${member.user} this channel has been claimed for your question. Please review <#${askHelpChannelId}> for how to get help.`,
 		);
-		await this.ensureAskChannels(msg.guild);
+		await this.ensureAskChannels(msg.guild!);
```

Comment:
I think you only need `!` on the first reference.

Reply:
It's needed everywhere. Assertions don't narrow.